CRM/Civicase/DAO/CaseCategoryInstance.php

```diff
@@ -1,14 +1,12 @@
 <?php
 
 /**
- * Civicrm DAO File.
- *
```

Comment:
Why is this removed? Maybe it's better to write some one-line description here?

Reply:
@ivan-compucorp This is autogenerated by Civix.